model_fitting/module_wraper.py

```diff
@@ -305,7 +305,7 @@ def build_classifier(reads_path, motifs_path, model_path, logs_dir, sample2bc, n
 
 def get_faa_file_name_from_path(path, use_mapitope):
     for file_name in os.listdir(path):
-        if file_name.endswith('faa') and 'unique' not in file_name and ('mapitope' in file_name) == use_mapitope:
+        if file_name.endswith('faa') and 'unique' in file_name and ('mapitope' in file_name) == use_mapitope:

```

Comment:
Shouldn't we have a flag to support previous code in case RPM implementation has issues?